extensions/amp-story-dev-tools/0.1/amp-story-dev-tools.css

```diff
@@ -18,10 +18,85 @@ amp-story-dev-tools {
   width: 100% !important;
   height: 100% !important;
   box-sizing: border-box !important;
-  background-color: #121212 !important;
+  background-color: black !important;
   font-size: 16px !important;
   font-family: 'Poppins', sans-serif !important;
   color: white !important;
   padding: 30px 10% !important;
-  text-align: center !important;
+  --i-amphtml-story-dev-tools-accent: #79B3FF;
+  --i-amphtml-story-dev-tools-accent-translucent: #79B3FF55;
+}
+
+.i-amphtml-story-dev-tools-header {
+  display: flex !important;
+  align-items: center !important;
+  justify-content: space-between !important;
+}
+
+.i-amphtml-story-dev-tools-brand {
+  display: flex !important;
+}
+
+.i-amphtml-story-dev-tools-brand-text > span {
+  display: block !important;
+  letter-spacing: 0.1em !important;
+  line-height: 1.2em !important;
+  padding-left: 10px !important;
```

Comment:
Let's add RTL support it's very easy to start now, it'll get harder later. RTL stories would show the devtools in a weird way otherwise.

Reply:
Added `padding-inline-start` and `margin-inline-start` instead of these things to make everything work well with rtl